commands/publish/README.md

```diff
@@ -62,6 +63,15 @@ This is useful when a previous `lerna publish` failed to publish all packages to
 - [`--yes`](#--yes)
 - [`--tag-version-prefix`](#--tag-version-prefix)
 
+### `--_auth`
```

Comment:
This needs to be named `--legacy-auth`

Reply:
> I'm half-inclined to _only_ allow this config via `lerna.json`, avoiding the CLI bloat. Thoughts?

This won't work for our teams that need to pass in the auth in CI. I didn't think it added a lot of CLI bloat, it was just one flag, since the options are passed to npm publish. But I understand the maintenance concern of adding *another flag* to this command.

Reply:
Ah, yes, understood.